src/components/ArticlePage/MarkdownHtml.js

```diff
@@ -89,8 +89,15 @@ const Html = styled.div`
   }
   h2 {
     ${H4.styles}
-    margin: ${2 * baselineSpacing}px 0 ${baselineSpacing}px 0;
+    margin: ${1.5 * baselineSpacing}px 0 ${baselineSpacing}px 0;
```

Comment:
You should rather use a multiple of the `baselineSmall` variable instead of calculating something from the `baselineSpacing`, that is just a quick shorthand for cases where I wasn't sure what to put there.

```suggestion
    margin: ${6 * baselineSmall}px 0 ${baselineSpacing}px 0;
```